code/modules/mob/living/carbon/human/life.dm

```diff
@@ -917,9 +917,8 @@
 	if(isSynthetic())
 		return
 
-	if(reagents.has_reagent("formaldehyde")) //embalming fluid stops decay
+	if(reagents.has_reagent("formaldehyde") || has_trait(TRAIT_HUSK)) //embalming fluid stops decay

```

Comment:
Not sure why this applies to husks. Hm. I'll be making various notes for things I want to consider later.